.github/workflows/build-and-push-cliain.yaml

```diff
@@ -1,9 +1,11 @@
+---
 name: Build and push cliain docker image
 
 on:
   push:
     paths:
       - 'bin/cliain/**'
+      - .github/workflows/build-and-push-cliain.yaml
```

Comment:
it would be perfect if GH exposes some variable encoding the current file, so that we have this entry the same in all workflow files (and be resistant to file name changes)

Reply:
The closest match is this one `github.action_path`, although it's not what we need here, as it is available only in composite actions
> The path where an action is located. This property is only supported in composite actions. You can use this path to access files located in the same repository as the action, for example by changing directories to the path: cd ${{ github.action_path }} .